lib/IDE/CodeCompletion.cpp

```diff
@@ -2116,7 +2049,7 @@ class CompletionLookup final : public swift::VisibleDeclConsumer {
   void addVarDeclRef(const VarDecl *VD, DeclVisibilityKind Reason) {
     if (!VD->hasName() ||
         !VD->isAccessibleFrom(CurrDeclContext) ||
-        shouldHideDeclFromCompletionResults(VD))
+        VD->shouldHideFromEditor())
```

Comment:
Is the function `shouldHideDeclFromCompletionResults`  removed? I can't find it in this patch.